src/components/TeamFounder/FounderProfileDescription.tsx

```diff
@@ -0,0 +1,20 @@
+import { ProfileDescriptionDiv } from "./FounderProfileDescription.styles";
```

Comment:
Duplicate of src/components/TeamSection/ProfileDescription.tsx

Reply:
Yes, but with the addition of props.description on line 6 & 15

Reply:
Because the Founder profile and the description are going to be side by side, I'll keep FounderProfileDescription.styles